tests/test_ext.py

```diff
@@ -14,41 +15,51 @@
 }
 
 
-@contextfunction
-def gettext(context, msgid):
-    language = context.get('LANGUAGE', 'en')
-    rv = languages.get(language, {}).get(msgid, msgid)
-    if context.eval_ctx.autoescape:
-        rv = Markup(rv)
-    return rv
+# Note: This isn't thread-safe, so if we ever decide to parallelize this
+# we'll need to change it to a thread local or something like that.
+_lang = 'en'
+
+
+@contextmanager
+def override_language(lang):
+    global _lang
+    try:
+        _lang = lang
+        yield
+    finally:
+        _lang = 'en'
```

Comment:
No, because I need to change the `_lang` in this module because it's what affects the fake gettext system we have here. I did model the override mechanics after the Django one.